HATE/HATE.csproj

```diff
@@ -1,56 +1,28 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Eto.Forms/2.7.0">
 
-  <PropertyGroup>
-    <OutputType>WinExe</OutputType>
-    <TargetFramework>net6.0-windows7.0</TargetFramework>
-    <UseWindowsForms>true</UseWindowsForms>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+  <!--
+    Set the BuildPlatform property to the Eto platform you wish to build for.
+    The default is the platform you are building on.
+    Valid values: Wpf, Windows, Mac64, XamMac2, Gtk, Direct2D
+  -->
 
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
     <RootNamespace>HATE</RootNamespace>
-    <AssemblyName>HATE</AssemblyName>
-    <ApplicationIcon>icon.ico</ApplicationIcon>
+    <AssemblyName>HATE-UML</AssemblyName>
```

Comment:
I think I will keep the old assembly name for the funnies - will do this after merging

Reply:
I can also change that if you want to.